demos/simple-menu.html

```diff
@@ -81,58 +109,126 @@
     </header>
 
     <main>
-      <div class="mdc-toolbar-fixed-adjust"></div>
-      <section class="hero">
```

Comment:
Why was the hero section removed? If this was for button positioning, shouldn't it be possible to work within the `demo-content` element for that and leave the hero section intact?

Reply:
brought it back. Still need to figure out how to size demo section correctly.

Reply:
added it back. Ignoring issues with height of the demo content portion.